packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.css

```diff
@@ -0,0 +1,89 @@
+.root {
+    display: grid;
+    gap: 2em;
+}
+
+.apply_button {
+    composes: root_normalPriority from '../../Button/button.css';
+
+    /* match the height of the input form. */
+    height: 2.25rem;
+    width: 2rem;
+} 
+
+.balance {
+    align-items: start;
+    display: grid;
+    grid-template-columns: max-content 1fr;
+    font-style: italic;
+}
+
+.card {
+    display: grid;
+    gap: 1rem;
+}
+
+.card_input_container {
+    align-items: center;
+    display: grid;
+    grid-template-columns: 1fr min-content min-content;
+    gap: 1em;
+}
+
+.card_input {
+    composes: root from '../../Field/fieldIcons.css';
+    width: 100%;
```

Comment:
https://github.com/magento/pwa-studio/pull/2123/files#diff-a77804cf0b1634bdd0e60b2ae8576eaeR7

![](https://i.kym-cdn.com/entries/icons/original/000/023/397/C-658VsXoAo3ovC.jpg)

Reply:
I think the right way to approach this is to just apply the width to the `FieldIcon` component. We should be in the practice of "fixing" them when they don't quite meet our needs anymore. Reach for customization only after concluding that it's not something we can apply generally. In this particular case, it is :)

Reply:
Fixed in https://github.com/magento/pwa-studio/pull/2124/commits/e70ba23133ae0d7f6339b39488105bebe3df9e6a.